app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -12,29 +12,58 @@ def index
 
       protected
 
+      def api_status_from_error(err)
+        if err.instance_of?(Faraday::ClientError)
+          :bad_request
+        elsif err.instance_of?(Faraday::ConnectionFailed)
+          :bad_gateway
+        elsif err.instance_of?(Faraday::TimeoutError)
+          :gateway_timeout
+        else
+          :internal_server_error
+        end
+      end
```

Comment:
I missed the PR that this came from but we can also use `case / when` here because `===` on classes checks instanceof
```suggestion
      def api_status_from_error(err)
        case err
        when Faraday::ClientError
          :bad_request
        when Faraday::ConnectionFailed
          :bad_gateway
        when Faraday::TimeoutError
          :gateway_timeout
        else
          :internal_server_error
        end
      end
```